src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/batch_preparers.py

```diff
@@ -26,14 +28,14 @@ def create_resource(self, name, **kwargs):
             self.resource_group = self._get_resource_group(**kwargs)
             cmd = 'az batch account create -n {} -g {} -l {}'.format(name, self.resource_group,
                                                                      self.location)
-            execute(cmd)
+            execute(self.cli_ctx, cmd)
```

Comment:
Do these pass?  In the compute tests, I had to pass a command object to test the custom commands. The surface level commands don't take cli_ctx directly.

Reply:
hmm interesting... yes they're passing for me locally... I'll double check...

Reply:
@tjprescott - I based it on what the SDK preparers were doing:
https://github.com/Azure/azure-cli/blob/knack-integration/src/azure-cli-testsdk/azure/cli/testsdk/preparers.py#L40

Reply:
If they are passing, then it's all good. :)